RELEASES.md

```diff
@@ -1,3 +1,142 @@
+Version 1.52.0 (2021-04-29)
+============================
+
+Language
+--------
+- [Added the `unsafe_op_in_unsafe_fn` lint, which checks whether the unsafe code
+  in an `unsafe fn` is wrapped in a `unsafe` block.][79208] By default this lint
+  is allow by default, and may become a warning or hard error in a
+  future release.
+- [You can now cast mutable references to arrays to a pointer of the same type as
+  the element.][81479]
+
+Compiler
+--------
+- [Upgraded the default LLVM to LLVM 12.][81451]
+
+Added tier 3\* support for the following targets.
+
+- [`s390x-unknown-linux-musl`][82166]
+- [`riscv32gc-unknown-linux-musl` & `riscv64gc-unknown-linux-musl`][82202]
+- [`powerpc-unknown-openbsd`][82733]
+
+Libraries
+---------
+- [`OsString` now implements `Extend` and `FromIterator`.][82121]
+- [`cmp::Reverse` now has `#[repr(transparent)]` representation.][81879]
+- [`Arc<impl Error>` now implements `error::Error`.][80553]
+- [All stable methods on integers are now `const`.][80962]
```

Comment:
This is not exactly right: `from_str_radix` is not `const`. (#80962 stabilized const for all stable methods that were already unstably-const, which was all remaining stable methods except for `from_str_radix`.)